.eslintignore

```diff
@@ -1,18 +1,19 @@
 /.git
-/docs/.next
 /coverage
+/docs/.next
 /docs/export
 /examples/create-react-app*/src/serviceWorker.js
 /examples/gatsby/public/
 /examples/preact/config
 /examples/preact/scripts
+/framer/Material-UI.framerfx/design/document.json
```

Comment:
What's the reason this shouldn't be formatted/linted?

Reply:
This file is machine-generated by Framer. It's not meant to be read by humans.